garden/tutorials/spawn_urdf/rrbot.urdf

```diff
@@ -0,0 +1,312 @@
+<?xml version="1.0" ?>
+<!-- =================================================================================== -->
+<!-- |    This document was autogenerated by xacro from rrbot.xacro                    | -->
+<!-- |    EDITING THIS FILE BY HAND IS NOT RECOMMENDED                                 | -->
+<!-- =================================================================================== -->
+<!-- Revolute-Revolute Manipulator -->
+<robot name="rrbot">
+  <!-- Space btw top of beam and the each joint -->
+  <!-- ros_control plugin -->
+  <gazebo>
+    <plugin filename="libgazebo_ros_control.so" name="gazebo_ros_control">
```

Comment:
Now that you mention it, I hadn't checked the cli output :grimacing:. I had only checked that the behaviour was as expected in the Gazebo GUI. 

I tested the tutorial on Ubuntu 20.04 LTS and Gazebo Fortress (from sources because I'm using it with ros noetic). 

There are a few errors thrown. 
```
[Err] [SystemLoader.cc:78] Failed to load system plugin [libgazebo_ros_control.so] : couldn't find shared library.
[Err] [SystemLoader.cc:78] Failed to load system plugin [libgazebo_ros_camera.so] : couldn't find shared library.
[Err] [SystemLoader.cc:78] Failed to load system plugin [libgazebo_ros_gpu_laser.so] : couldn't find shared library.
[GUI] [Err] [SystemPaths.cc:378] Unable to find file with URI [model://rrbot_description/meshes/hokuyo.dae]
[GUI] [Err] [SystemPaths.cc:473] Could not resolve file [model://rrbot_description/meshes/hokuyo.dae]
[GUI] [Err] [MeshManager.cc:173] Unable to find file[model://rrbot_description/meshes/hokuyo.dae]
[GUI] [Err] [MeshDescriptor.cc:56] Mesh manager can't find mesh named [model://rrbot_description/meshes/hokuyo.dae]
[GUI] [Err] [Ogre2MeshFactory.cc:562] Cannot load null mesh [model://rrbot_description/meshes/hokuyo.dae]
[GUI] [Err] [Ogre2MeshFactory.cc:125] Failed to get Ogre item for [model://rrbot_description/meshes/hokuyo.dae]
[GUI] [Err] [SceneManager.cc:394] Failed to load geometry for visual: link3_fixed_joint_lump__hokuyo_link_visual_2
```

Should this URDF be replaced by the one in your branch? https://github.com/ros-simulation/gazebo_ros_demos/tree/ahcorde/port/ros2

Reply:
I tested it with the urdf from the .xacro from your branch @ahcorde 

The number of errors is now reduced to 3.
```
[Err] [SystemLoader.cc:78] Failed to load system plugin [libgazebo_ros_control.so] : couldn't find shared library.
[Err] [SystemLoader.cc:78] Failed to load system plugin [libgazebo_ros_camera.so] : couldn't find shared library.
[Err] [SystemLoader.cc:78] Failed to load system plugin [libgazebo_ros_gpu_laser.so] : couldn't find shared library.
```